plugin/pkg/scheduler/core/generic_scheduler_test.go

```diff
@@ -410,7 +410,7 @@ func TestHumanReadableFitError(t *testing.T) {
 			"3": []algorithm.PredicateFailureReason{algorithmpredicates.ErrNodeUnderDiskPressure},
 		},
 	}
-	if strings.Contains(error.Error(), "No nodes are available that match all of the following predicates") {
+	if strings.Contains(error.Error(), NoNodeAvailableMsg) {
```

Comment:
Can you also help to remove `:` in `NoNodeAvailableMsg`? It's unnecessary to me :).